src/app-sync.js

```diff
@@ -7,12 +7,12 @@ import {
   requestLoggerMiddleware,
   validateUserMiddleware,
 } from './util/middlewares.js';
-import getAccountDb from './account-db.js';
 import { getPathForUserFile, getPathForGroupFile } from './util/paths.js';
 
 import * as simpleSync from './sync-simple.js';
 
 import { SyncProtoBuf } from '@actual-app/crdt';
+import getAccountDb, { isAdmin } from './account-db.js';
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏I believe most of the endpoints in this file should be moved behind a permission check. For example: I should not be able to `/sync` a file that I do not have permissions to.

Ideally we should do this in a middleware so it could be bulk-applied to all the relevant endpoints.